ui/app/AppLayouts/Wallet/controls/StatusNetworkSelector.qml

```diff
@@ -193,7 +193,7 @@ Rectangle {
 
                     asset.height: root.asset.height
                     asset.width: root.asset.width
-                    asset.name: root.useLetterIdenticons ? model.text : Style.svg(model.iconUrl)
+                    asset.name: root.useLetterIdenticons ? model.text : (model.isTest ? Style.svg(model.iconUrl + "-test") : Style.svg(model.iconUrl))
```

Comment:
@stefandunca Do you think it would be an option to update the status-go iconUrl field for each testnet? It will probably affect mobile as well. Would be nice to remove this kind of logic.

Reply:
Oh yes please